src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -352,6 +359,33 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
             dataImportBatch.fields[DATA_IMPORT_BATCH_ID_INFO.fieldApiName] = this.recordId;
         }
 
+        dataImportBatch = this.stripNamespaceFromDonationMatchingRuleFields(dataImportBatch);
+
+        return dataImportBatch;
+    }
+
+    /***************************************************************************
+     * @description Strips namespace prefix from the Data Import Batch donation matching
+     * rule fields if in a namespaced context
+     * @param dataImportBatch
+     * @returns {object}
+     */
+    stripNamespaceFromDonationMatchingRuleFields (dataImportBatch) {
+        const donationMatchingRule = dataImportBatch.fields[
+          DATA_IMPORT_BATCH_DONATION_MATCHING_RULE.fieldApiName];
+        const namespace = getNamespace(DATA_IMPORT_BATCH_INFO.objectApiName);
+        
+        if (isNull(namespace)) {
+            let strippedFields = '';
+            const matchingRuleFields = donationMatchingRule.split(COMMA_SEPARATOR);
+            matchingRuleFields.forEach(field => {
+                strippedFields += stripNamespace(field,
```

Comment:
I'm a bit confused here - since we are inside the `if (isNull(namespace))` block here, isn't the value of `namespace` always going to be null?  And as a result, is the call to stripNamespace() going to actually do anything? 

I did some console logging in a non-namespaced scratch org and got these values: 
- Namespace: null
- Matching rule fields: npsp__Donation_Amount__c,npsp__Donation_Date__c
- Stripped Fields: npsp__Donation_Amount__c;npsp__Donation_Date__c

Running through the steps in the WI, The batch seems to process successfully, but it looks like the value for the donation matching rule isn't actually being changed at all. Am I missing something? 

@voduyemi @psadie-sfdo

Reply:
@ptewson-sfdo You're right, I was referencing an already `null` namespace variable, I should be passing in the `npsp__` string. It's fixed now.

Reply:
@ptewson-sfdo The batches will still get process as long as we're not trying to match donations (triggering the review donations modal). That's the reason why you were still able to process/dry run batches.